docs/src/main/asciidoc/client-customization.adoc

```diff
@@ -0,0 +1,47 @@
+== Client Customization
+
+The library uses the https://github.com/Azure/azure-sdk-for-java[Azure SDK for Java] for connecting to Azure App Configuration and Azure Key Vault. Two interfaces, `ConfiguraitonClientBuilderSetup` and `SecretClientBuilderSetup` are provided to modify the clients. Each interface has a `setup` method that takes in their respective builder along with the `String` value of the URI that the client will connect to. 
```

Comment:
Which library? Specify if this for App Config, otherwise this can get confusing because you have mentioned multiple libraries previously in the article

I think you mean ConfigurationClientBuilderSetup